packages/cascara/src/modules/ActionEdit/ActionEdit.js

```diff
@@ -102,44 +102,38 @@ const ActionEdit = ({
 
   return isEditing ? (
     <>
```

Comment:
Removing Tooltip right now as it appears there is a memory leak in the delay on showing the tooltip. We will need to define a cleanup useEffect hook.